src/components/JBrowse/JBrowseView.js

```diff
@@ -13,17 +13,33 @@ const defaultFooterStyles = {
 const theme = createJBrowseTheme();
 
 const assembly = {
-  name: 'GRCh38',
+  name: 'hg19',
+  aliases: ['GRCh37'],
   sequence: {
     type: 'ReferenceSequenceTrack',
-    trackId: 'GRCh38-ReferenceSequenceTrack',
+    trackId: 'Pd8Wh30ei9R',
```

Comment:
Is there a way we can pass this as config tooo?

Reply:
We definitely can. But, we don't need to do that as we are supporting only one reference track. We will get a new user story and requirements soon